src/rules/javascript/index.js

```diff
@@ -0,0 +1,25 @@
+import content_scripts_file_absent from './content-scripts-file-absent';
+import deprecated_entities from './deprecated-entities';
+import event_listener_fourth from './event-listener-fourth';
+import global_require_arg from './global-require-arg';
+import opendialog_nonlit_uri from './opendialog-nonlit-uri';
+import opendialog_remote_uri from './opendialog-remote-uri';
+import webextension_api from './webextension-api';
+import webextension_api_compat from './webextension-api-compat';
+import webextension_api_compat_android from './webextension-api-compat-android';
+import webextension_deprecated_api from './webextension-deprecated-api';
+import webextension_unsupported_api from './webextension-unsupported-api';
+
+export default {
+  'content-scripts-file-absent': content_scripts_file_absent,
+  'deprecated-entities': deprecated_entities,
+  'event-listener-fourth': event_listener_fourth,
+  'global-require-arg': global_require_arg,
+  'opendialog-nonlit-uri': opendialog_nonlit_uri,
+  'opendialog-remote-uri': opendialog_remote_uri,
+  'webextension-api': webextension_api,
+  'webextension-api-compat': webextension_api_compat,
+  'webextension-api-compat-android': webextension_api_compat_android,
+  'webextension-deprecated-api': webextension_deprecated_api,
+  'webextension-unsupported-api': webextension_unsupported_api,
+};
```

Comment:
This looks almost like content we should generate...

Do we have test coverage otherwise? (thinking about preventing missing/forgotten imports and/or new rules not registered here)

Reply:
yes, initially I was also considering to auto-generate it, but to be honest we didn't changed the list of rules that often and so I didn't bothered for now, and instead I opted to cover it with an additional test case for now (and defer generating it automatically to when it may become useful or necessary in the future).